core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/ai-content-prompt.component.ts

```diff
@@ -0,0 +1,64 @@
+import { of } from 'rxjs';
+
+import { Component, ElementRef, EventEmitter, Output, ViewChild } from '@angular/core';
+import { FormBuilder, FormGroup, FormControl, Validators } from '@angular/forms';
+
+import { catchError } from 'rxjs/operators';
+
+import { AiContentService } from '../../shared/services/ai-content/ai-content.service';
+
+interface FormValues {
+    textPrompt: string;
+}
+
+@Component({
+    selector: 'dot-ai-content-prompt',
+    templateUrl: './ai-content-prompt.component.html',
+    styleUrls: ['./ai-content-prompt.component.scss']
+})
+export class AIContentPromptComponent {
+    @ViewChild('input') input: ElementRef;
+
+    @Output() formValues = new EventEmitter<FormValues>();
+    @Output() formSubmission = new EventEmitter<boolean>();
+
+    loading = false;
+    form = new FormGroup({
+        textPrompt: new FormControl('')
+    });
+
+    constructor(private fb: FormBuilder, private aiContentService: AiContentService) {
+        this.buildForm();
+    }
+
+    onSubmit() {
+        const textPrompt = this.form.value.textPrompt;
+
+        if (textPrompt) {
+            this.formValues.emit({ textPrompt });
+
+            this.formSubmission.emit(true);
+
+            this.aiContentService
+                .getIAContent(textPrompt)
+                .pipe(catchError(() => of(null)))
+                .subscribe((response) => {
+                    console.warn('AI response', response);
+                });
```

Comment:
When the extension isn't closed after submitting a prompt, it can take 2-3 seconds, or longer for larger responses, to receive feedback. Thus, users might not see any response for several seconds after submission.

So, my understanding was that should be closed immediately.

Reply:
@fmontes what do you think about this case? Is there a better `UI/UX`?

Reply:
@rjvelazco @fmontes I just spoke with @john-thomas-dotcms about this. 

- You were right about the extension not being closed until a response is there, The idea is to add a "pending" label instead of the "send" icon while we waiting for a response. 

Will update it the PR